### PR TITLE
update project docs for dcp

### PR DESCRIPTION
**Pull request by sramanan**
@nchilada @tscanlin @aliabbasrizvi @ajithopti - please read

### CODE REVIEW

File: src/content/rest/reference/projects/2-create.md [hunk 1]
```diff
@@ -25,7 +25,8 @@ response: |
     "exclude_names": null,
     "ip_anonymization": false,
     "ip_filter": "",
-    "socket_token": "AABBCCDD~123456789"
+    "socket_token": "AABBCCDD~123456789",
+    "dcp_service_id": null
```
Comment (nchilada) [inline src/content/rest/reference/projects/2-create.md]: Maybe make this `121234` as well?

**Comment by nchilada** [comment]
LGTM

File: src/content/rest/reference/projects/2-create.md (hunk 1 above)
Comment (aliabbasrizvi) [inline src/content/rest/reference/projects/2-create.md]: +1

Comment (aliabbasrizvi) [inline src/content/rest/reference/projects/2-create.md]: Synced with @sramanan ... this is expected

Comment (nchilada) [inline src/content/rest/reference/projects/2-create.md]: Oh, I see.  A new project can't have a DCP service attached to it yet.

**Pull request merged**